common/_variables.scss

```diff
@@ -3,6 +3,10 @@ $main-color: #ff8000 !default;
 $link-color: #126f9a !default;
 $header-bg: #fff !default;
 $footer-bg: #fff !default;
+$header-btn-bg: $header-bg !default;
+$header-btn-bottom-border-width: 4px !default;
+$header-btn-bottom-border-hover-color: theme-color("inverse") !default;
+$header-course-link-btn-bottom-border-color: $link-color !default;
```

Comment:
Please add the new variable in the [README](https://github.com/open-craft/edx-simple-theme/blob/master/README.md)